CMakeLists.txt

```diff
@@ -387,6 +428,16 @@ target_include_directories(${BIN_TARGET} PRIVATE
   3rdParty/Radon/Radon/include
   3rdParty/libsmacker)
 
+if(DISCORD_INTEGRATION AND WIN32)
+  target_include_directories(${BIN_TARGET} PRIVATE "${discordsrc_SOURCE_DIR}/..")
+
+  set(DISCORD_LIB_DIR "${discordsrc_SOURCE_DIR}/lib/x86_64")

```

Comment:
`${CMAKE_HOST_SYSTEM_PROCESSOR}` instead of `x86_64`, so that it works for the 32-bit BUILD as well

Reply:
`${CMAKE_HOST_SYSTEM_PROCESSOR}` prints AMD64 on both x86 and x64 configurations in VS. Will fix CMake for all platforms eventually since this is still a WIP.

Reply:
Oh, it's a configuration in VS which means we probably have to use a generator expression

I've found this answer from 2017 but hopefully there is a better solution these days https://cmake.org/pipermail/cmake-developers/2017-November/030451.html